src/harness/harness.ts

```diff
@@ -1244,6 +1247,14 @@ namespace Harness {
 
             const docs = inputFiles.concat(otherFiles).map(documents.TextDocument.fromTestFile);
             const fs = vfs.createFromFileSystem(IO, !useCaseSensitiveFileNames, { documents: docs, cwd: currentDirectory });
+            if (symlinks) {

```

Comment:
If `symlinks` were a `vfs.FileSet`, you could just call `fs.apply(symlinks)`.

Reply:
Then I'd have to distinguish between weather the link is between files or directories, which I'm not bothering to do right now, as it really doesn't matter. It's be a lot more code to do that same thing 🤷‍♀️